Tribler/Category/Category.py

```diff
@@ -89,14 +90,8 @@ def calculateCategory(self, torrent_dict, display_name):
         return self.calculateCategoryNonDict(files_list, display_name, tracker, comment)
 
     def calculateCategoryNonDict(self, files_list, display_name, tracker, comment):
-        # Check xxx
-        try:
-
-            if self.xxx_filter.isXXXTorrent(files_list, display_name, tracker, comment):
-                return 'xxx'
-        except:
-            self._logger.critical(
-                'Category: Exception in explicit terms filter in torrent: %s', display_name, exc_info=True)
```

Comment:
I am not sure what you mean by returning None, but the [ function](https://github.com/Tribler/tribler/blob/3ba145c31e79d886b27e6ab4d426217c86ec6e4f/Tribler/Category/FamilyFilter.py#L45) doesn't seem to raise stuff. Only that a[0] is scary, but if that happens we are passing wrong arguments to the function (imo) anyway.